### ISSUE TITLE
dbatools install broken on some Standard Edition

### ISSUE THREAD

**Issue by bflippin** [marcingminski/sqlwatch#482]
**Did you check DOCS to make sure there is no workaround?**
https://sqlwatch.io/docs/

**Describe the bug**

Running install through DBAtools on SQL Server 2014 Standard fails due to compression being on by default.

**To Reproduce**
Run the dbatools installer on any edition of SQL Server that doesn’t support compression.

**Expected behavior**

I would like either an edition check or a command line switch to turn on/off compression instead of the install failing.

**Screenshots**

**Windows Server (please complete the following information):**
 - OS Version: Windows 2012 R2

**SQL Server (please complete the following information):**
 - SQL Version: SQL Server 2014
 - SQL Edition: Standard
 
**SQL Server Management Studio (SSMS -> about -> copy info): Did not use SSMS here.
 
**SQLWATCH version (from DACPAC or from sysinstances)**
 - 1.0.0.0
 
**Additional context**

If this should go to the dbatools team instead I will submit it over there. I was able to remove the compression defaults and deploy without issue from Visual Studio.

**Comment by marcingminski** [comment]
Yeah that was changed in 4.8 on the basis that it will make life easier for the majority running 2016 onwards. Apologies it didn’t work for you. Being able to pass parameters during install would be ideal, but impossible with the current implementation (compression is define in the project as part of Schema).

I guess documentation would need updating.

Anyway, happy to help you get this installed on 2014 if you didn’t manage yet .
Or, if you are familiar with database projects, you’d need to remove the compression from the table definition (quite a lot of find and replace), rebuild the project and deploy.
